src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -210,7 +210,6 @@ public static bool IsLicensed(Type type)
             if (license != null)
             {
                 license.Dispose();
-                license = null;
             }
```

Comment:
This can now just be `license?.Dispose();`.

Reply:
Changed